Assets/SEE/DataModel/DG/StandardNames.cs

```diff
@@ -51,4 +51,121 @@ public static string Name(this NumericAttributeNames numericAttributeName)
             };
         }
     }
-}
\ No newline at end of file
+
+    /// <summary>
+    /// Defines names of JaCoCo code-coverage metrics.
+    /// </summary>
+    public static class JaCoCo
+    {
+        /// <summary>
+        /// Names of node attributes provided by JaCoCo.
+        /// See also: https://www.eclemma.org/jacoco/trunk/doc/counters.html
+        /// </summary>
```

Comment:
This documentation comment is free-floating

Reply:
Fixed it.